packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -76,7 +82,8 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
         errorMessage={errorMessage}
         validationState={validationState}
         isDisabled={isDisabled}
-        showErrorIcon={showErrorIcon} />
+        showErrorIcon={showErrorIcon}
+        gridArea="helpText" />
```

Comment:
surprised this isn't set on the css class you pass on slots

Reply:
I don't think there is a slot for help text

Reply:
tch, you're right, i read it as contextual help in my head, too many helps